cmd/antrea-agent/options.go

```diff
@@ -254,10 +257,19 @@ func (o *Options) validateFlowExporterConfig() error {
 				return fmt.Errorf("IdleFlowExportTimeout is not provided in right format")
 			}
 			if o.idleFlowTimeout < o.pollInterval {
-				o.activeFlowTimeout = o.pollInterval
+				o.idleFlowTimeout = o.pollInterval
 				klog.Warningf("IdleFlowExportTimeout must be greater than or equal to FlowPollInterval")
 			}
 		}
+		if (o.activeFlowTimeout > defaultStaleConnectionTimeout) || (o.idleFlowTimeout > defaultStaleConnectionTimeout) {
+			if o.activeFlowTimeout > o.idleFlowTimeout {
+				o.staleConnectionTimeout = 2 * o.activeFlowTimeout
+			} else {
+				o.staleConnectionTimeout = 2 * o.idleFlowTimeout
+			}
+		} else {
+			o.activeFlowTimeout = defaultStaleConnectionTimeout
```

Comment:
that doesn't seem right to me, I don't remember seeing in previous versions of the PR and I don't think we should override the user-provided value

Reply:
It is a mistake from my end. Used a wrong variable name :)
Wanted to initialize the stale connection timeout.

Because of that, I see e2e failures.. resolving this fixes those.